sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -225,8 +278,20 @@ def get_queue_sender(self, queue_name, **kwargs):
         self._handlers.add(handler)
         return handler
 
-    def get_queue_receiver(self, queue_name, **kwargs):
-        # type: (str, Any) -> ServiceBusReceiver
+    def get_queue_receiver(
+        self,
+        queue_name: str,
+        *,
+        session_id: Union[str, NEXT_AVAILABLE_SESSION] = None,
```

Comment:
why is NEXT_AVAILABLE_SESSION available in the public API, rather than `ServiceBusSessionFilter`. I know it's the only option for now, but it seems strange.

If we only want that constant to be available, rather than any other future values added to `ServiceBusSessionFilter`, should the type be `Union[str, int]`? And, since None is an option, shouldn't the type here be `Optional[Union[str, int]]`?

Reply:
you're right... it's a bit strange here as `NEXT_AVAILABLE_SESSION` is a constant value not a class. Also, ServiceBusSessionFilter is not an extensible Enum so it's not of type int, Union[str, int] wouldn't work unless we make it an extensible Enum -- `class ServiceBusSessionFilter(int, Enum)`.

I feel that the right approach is to expose `ServiceBusSessionFilter`, however, I feel it's overkilling to expose an Enum with just one member.

 how about let's keep it for now unless someone explains?

Reply:
I'm def with you on that. Unfortunately, this Enum is not a valid type, so I get a mypy error. Maybe I can just try to ignore the type? Not sure if that would work, but is that preferable?

Reply:
remove explicit kwarg for now, if resolution not reached